.github/workflows/SBOM.yaml

```diff
@@ -0,0 +1,35 @@
+name: Generate SBOM
+
+on:
+  push:
+    
+
+jobs:
+  generate_sbom:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2  #get depo code
+      
+      # SBOM generator JSON
+      - name: Generate Python SBOM
+        uses: CycloneDX/gh-python-generate-sbom@v2
+        with:
+          input: ./requirements.txt
```

Comment:
Is the relative path absolutely needed? Normally you will be by default at the root of the repository